### ISSUE TITLE
Request: Filtering by multiple criteria

### ISSUE THREAD

**Issue by matijaerceg** [theypsilon/_arcade-organizer#51]
The thought is "I wish I could just see all the games that work on my CRT (consumer, NTSC only), not rotated, using button controller only"

Basically, I'm wishing I could tell the script to create a couple of additional subfolders that are based on multiple (existing) criteria. Some examples:

1. 15khz, not-rotated, 8-way input, ntsc
2. 15khz, rotated left, 4-way input, ntsc and pal
3. 2-players, 8-way input

etc.

Is there any chance something like this might be possible in the future?

**Comment by theypsilon** [comment]
This is already possible, there is a UI for this in Update All Settings Screen under the Arcade Organizer submenu.

Go there and select 15khz video resolution, and only the rotation you are interested in. Same for buttons.

**Comment by matijaerceg** [comment]
What I'm hoping to do is keep all the titles in the usual folders, but also add some new custom folders, each one containing only the titles that match the filters in the example above. So I'd have:

- All the usual folders
- One new folder that I could call "8-way CRT"
- One new folder that I could call "4-way CRT"
- One new folder that I could call "2 players 8-way"
- etc

As I understand it, currently, whatever filters I choose for the script, that's all I'm going to get. If I want to filter by a new set of filters, I'd have to run the script again.

My use case is that sometimes I plug my MiSTer into my consumer CRT, sometimes I plug it into my TATE PVM, and sometimes I plug it into my HDMI TV for 2 player games. For each of those scenarios, I'd like to easily see what games work only in that scenario.

**Comment by theypsilon** [comment]
I feel that that could be too inefficient, since symlinks are a limiting factor for Arcade Organizer and the MiSTer menus. But thank you for your suggestion. If you think is worth exploring, you might try to work on a PR. Let me know if you need any help with that.

**Comment by matijaerceg** [comment]
I don't know what you mean about symlinks being a limiting factor. What I imagine would still just use symlinks. It's possible (and probable) that I'm explaining myself poorly 😅

**Comment by theypsilon** [comment]
The high volume of symlinks is affecting performance (AO runs slower). But maybe is not a big deal for the default configurations. Will you work on this task?

**Comment by matijaerceg** [comment]
I really wish I was able to.

Thank you for your feedback on my suggestion Jose. I'm sorry to only offer an 'ask' and no dev contribution. Feel free to close my issue of course.

Ultimately, I think that there's a great use case for being able to see a list of all arcade cores that fit the current situation. Not even necessarily for multiple setups, but even just one setup would be great. For example, if I go and look at all the 15khz games, that list doesn't tell me which games are rotated. If I look at the rotated list, that one doesn't tell me which games need 4-way stick. So even with the organizer, I would have to mentally keep a note to cross reference the lists. Am I being spoiled? Yes 😅

I'm going to do some digging to see if there's a linux script that lets me create a folder of 'overlapping' files that are found in multiple folders, as that would pretty much solve this use case for me.

**Comment by theypsilon** [comment]
No, thank you for opening this issue. It's a valid use case. I'll let this open in case somebody would like to help with the implementation.
